### ISSUE TITLE
Pipeline Values cannot be retrieved

### ISSUE THREAD

**Issue by leonardbinet** [circleci/circleci-docs#6355]
# Description

Following instructions detailed here https://circleci.com/docs/2.0/pipeline-variables/  I tried to retrieve the `pipeline.id` or `pipeline.number` values in my builds.

# Steps to Reproduce

Generate a basic circleci config:

```
version: 2

jobs:
  echo_stuff:
    docker:
      - image: cimg/node:17.4.0
    steps:
      - run: echo "This is pipeline ID << pipeline.id >>"

```

The output of such code would echo `"This is pipeline ID << pipeline.id >>"`.
Whereas I would have expected the actual id, ie `"This is pipeline ID 123"`.

<img width="1142" alt="Screenshot 2022-02-10 at 15 49 54" src="https://user-images.githubusercontent.com/1590371/153432426-cc0d3208-666c-4f95-9d46-aac286447053.png">

**Comment by rosieyohannan** [comment]
Hello @leonardbinet! Can you try the same config but using the latest config version of 2.1 and see if that gets you the correct output?

Pipelines are only supported with `version: 2.1` configuration. Thanks! And let me know how it goes :-)

**Comment by leonardbinet** [comment]
Thanks a lot @rosieyohannan , it works indeed. 

It might be worth it making it more visible in documentation though (the https://circleci.com/docs/2.0/pipeline-variables/  link refers to 2.0 version in URL), maybe adding an information panel stating that this feature is available from 2.1 on?

**Comment by rosieyohannan** [comment]
Good idea, I will look into adding something to this effect, thanks @leonardbinet :-D
